### ISSUE TITLE
Enable GitHub Sponsors

### ISSUE THREAD

**Issue by edumserrano** [cenfun/monocart-reporter#72]
Hi @cenfun,

I think the work you've done with `monocart-reporter` is awesome and I'd like to become your first Sponsor in hopes that many others will follow.

Please see:
- [Setting up GitHub Sponsors for your personal account](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account)
- [Displaying a sponsor button in your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository)

**Comment by cenfun** [comment]
![image](https://github.com/cenfun/monocart-reporter/assets/334577/9b834e66-72b2-40ee-b13a-97ca2a34cea4)
Unfortunately, I can't create my account in my region.

**Comment by edumserrano** [comment]
Well... that is very sad. I think you're doing amazing with this project. Perhaps in the future 🤞.

**Comment by michaelhays** [comment]
Completely agree! Will definitely contribute if and when you get a sponsorship program. Maybe https://ko-fi.com/?

**Comment by xyngfei** [comment]
I agree with you all. @cenfun has done a good job of maintaining this repo.

**Comment by agoldis** [comment]
@cenfun 

We at https://github.com/currents-dev would love to support your work. 

- are you open to sponsorship outside GitHub?
- may adding more maintainers from other regions can work... is there anyone you trust?

**Comment by cenfun** [comment]
@agoldis thanks, yes, but I currently don't have a bank account for that, and I don't know anyone else in other regions.

**Comment by shairez** [comment]
@cenfun have you tried https://opencollective.com/ ?
